packages/ui/src/components/va-time-picker/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -49,26 +50,18 @@ export default defineComponent({
 
     watch(syncActiveItemIndex, (newVal) => { scrollTo(newVal) })
 
-    onMounted(() => scrollTo(syncActiveItemIndex.value, false))
-
-    const scrollTo = (index: number, animate = true) => {
-      nextTick(() => {
-        const children = rootElement.value!.children
-
-        const element = children[index] as HTMLElement
+    onMounted(() => {
+      if (rootElement.value) {
+        // 170 here it's a sum of gaps from "before" and "after" pseudo elements
+        cellElementHeight.value = (rootElement.value!.scrollHeight - 170) / props.items.length
+      }
 
-        if (!element) {
-          rootElement.value?.scrollTo({
-            behavior: animate ? 'smooth' : 'auto',
-            top: 0,
-          })
-          return
-        }
+      scrollTo(syncActiveItemIndex.value)
+    })
 
-        rootElement.value?.scrollTo({
-          behavior: animate ? 'smooth' : 'auto',
-          top: element.offsetTop - element.parentElement!.offsetTop,
-        })
+    const scrollTo = (index: number) => {
+      nextTick(() => {
+        rootElement.value!.scrollTop = index * cellElementHeight.value
```

Comment:
We have lost cool scroll animation. Was it planned?

Reply:
Yep, on one of the iteration animation caused a lot of bugs. I'll try to restore.

Reply:
Yes, it's still buggy.